stories/ExampleSlider.jsx

```diff
@@ -129,8 +129,8 @@ storiesOf('Slider', module)
     );
   })
   .add('Dates', () => {
-    const startDate = new Date('01-01-2015').valueOf();
```

Comment:
This is also broken in Safari; another alternative is using slashes instead of dashes - maybe we should go with that?

Reply:
@ljharb I just fix date constructor (I forgot that dates start from 0), and seems like it's ok now:
![safari](http://joxi.ru/v29pdORCZwgzvm.jpg)